logo/skopeo.svg

```diff
@@ -0,0 +1,998 @@
+<?xml version="1.0" encoding="UTF-8" standalone="no"?>
```

Comment:
(I know, nothing like a clearly OK PR for bikeshedding…)

Consider using the `docs` directory, to have one less top-level item to deal with in directory listings, IDEs and the like.

Reply:
we keep logo in svg in cri-o `logo/` dir as well